src/Suave.Tests/Types.fs

```diff
@@ -57,6 +57,11 @@ let httpBinding (_ : SuaveConfig) =
       let binding = HttpBinding.create HTTP IPAddress.Loopback 80us
       let actual = binding.uri "" "" |> sprintf "%O"
       Expect.equal actual "http://127.0.0.1/" "uri"
+
```

Comment:
Wow, the current escaping behavior looks kind of wild, but I've added tests to describe it and to make sure my change didn't affect it. Changing it may break backwards compatibility and extend the scope of this PR so I'd rather avoid it for now.